WooCommerce/Classes/ViewRelated/Products/AI/ProductDescriptionGenerationView.swift

```diff
@@ -26,6 +26,8 @@ final class ProductDescriptionGenerationHostingController: UIHostingController<P
 struct ProductDescriptionGenerationView: View {
     @ObservedObject private var viewModel: ProductDescriptionGenerationViewModel
     @State private var copyTextNotice: Notice?
+    @State private var missingName: Bool = false
```

Comment:
Non-blocking opinion: I would prefer having these values in the view model so that it can also be unit tested.